apps/redi-connect/src/components/organisms/modules/MSessions.tsx

```diff
@@ -138,18 +138,21 @@ const MSessions = ({ sessions, menteeId, editable }: MSessions) => {
 
       {editable && (
         <Modal
+          styles={{ overflow: 'unset' }} // This is needed to be able to show the datepicker outside the modal
           show={showAddSession}
           stateFn={setShowAddSession}
           title="Log a new mentoring session"
         >
-          <Modal.Body>
+          {/** overflow: 'unset' is needed to be able to show the datepicker outside the modal */}
+          <Modal.Body style={{ overflow: 'unset' }}>
             {/* <Content>Please write a few words about why you feel uncertain about your mentorship and which issues you are experiencing? </Content> */}
```

Comment:
@helloanil can you push a hotfix removing this line? I don't see the utility of keeping this around. Seems related to the report-a-problem modal, it's got no purpose here.

Reply:
Done: https://github.com/talent-connect/connect/commit/ff43a43fbe599a99b5ee0764ef43342e56f33393

Reply:
Thanks!